frontend/openchat-client/src/openchat.ts

```diff
@@ -371,7 +371,7 @@ export class OpenChat extends EventTarget {
         }
 
         this.refreshAffectedEvents(chat, affectedEvents);
-        this.updateDetails(chat, this._liveState.events);
```

Comment:
We're safe to remove the events here since they will have already had their users processed when first loading them within `handleEventsResponse`.